app/views/medicaid/insurance_current_type/edit.html.erb

```diff
@@ -2,21 +2,28 @@
 
 <div class="form-card">
   <header class="form-card__header">
-    <div class="form-card__title">What type of insurance plan are you currently enrolled in?</div>
+    <div class="form-card__title">
+      <%= t("medicaid.insurance_current_type.edit.title",
```

Comment:
this is used for both single-member and multi-member. right now you can get a message that says "Tell us what insurance plan each person is enrolled in." even if only one member is effectively applying.

Reply:
(that comment is from me ^, and more a thing to be aware of but not change right now!)